EIPS/eip-2315.md

```diff
@@ -23,16 +23,38 @@ In the Appendix we show example solc output for a simple program that uses over
 
 ## Specification
 
+We introduce one more `stack` into the evm, called `return_stack`. The `return_stack` is limited to `1023` items.
```

Comment:
`stack` (in backticks) below refers to a data stack, and here it's just stack in general
```suggestion
We introduce one more stack into the EVM, called `return_stack`. The `return_stack` is limited to `1023` items.
```